packages/visual-editor/src/components/atoms/cta.tsx

```
@@ -84,10 +86,31 @@ const useResolvedCtaProps = (props: CTAProps) => {
         if (!props.presetImageType) {
           return null;
         }
+
+        let label = presetImageIcons[props.presetImageType];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)